tosp_auth/tests.py

```diff
@@ -1,3 +1,44 @@
-# from django.test import TestCase
+from django.core.urlresolvers import reverse
+from django.contrib.staticfiles.testing import StaticLiveServerTestCase
+from splinter import Browser
+from django.contrib.auth.models import User
 
-# Create your tests here.
+class WidgetLogoutTest(StaticLiveServerTestCase):
+    """
+        Integration test suite for testing the views in the app: tosp_auth.
```

Comment:
All the comments on docstrings should start on the first line, we also are following this documentation style the best we can: https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt#docstring-standard